### PR TITLE
[React-Native] Add missing props for `Text` & `TextInput`

### PR DESCRIPTION
**Pull request by jeremistadler**
- https://facebook.github.io/react-native/docs/text.html#selectioncolor
- https://facebook.github.io/react-native/docs/text.html#textbreakstrategy
- https://facebook.github.io/react-native/docs/textinput.html#spellcheck

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint react-native`

- [x] Provide a URL to documentation or source code which provides context for the suggested changes
- [x] Increase the version number in the header if appropriate.
- [x] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

File: types/react-native/index.d.ts [hunk 1]
```diff
@@ -1,6 +1,9 @@
 // Type definitions for react-native 0.44
 // Project: https://github.com/facebook/react-native
-// Definitions by: Eloy Durán <https://github.com/alloy>, Fedor Nezhivoi <https://github.com/gyzerok>, HuHuanming <https://github.com/huhuanming>
+// Definitions by: Eloy Durán <https://github.com/alloy>
+//                 Fedor Nezhivoi <https://github.com/gyzerok>
+//                 HuHuanming <https://github.com/huhuanming>
+//                 Jeremi Stadler <https://github.com/jeremistadler>
```
Comment (alloy) [inline types/react-native/index.d.ts]: Please keep it on one line, otherwise the bot will not be able to parse this header.

Comment (alloy) [inline types/react-native/index.d.ts]: And awesome for joining! 🎉

**Review by alloy — CHANGES_REQUESTED** [review]
Looks good, besides that header.

File: types/react-native/index.d.ts (hunk 1 above)
Comment (jeremistadler) [inline types/react-native/index.d.ts]: Thanks! Fixed the header! 🎉

**Comment by alloy** [comment]
👍

**Review by alloy — APPROVED** [review]

**Pull request merged**